doc/source/dev/data_types.md

```diff
@@ -0,0 +1,311 @@
+
+# Data Types
+
+## Adding a New Data Type
+
+**Note:** This is the 'old' method for doing this. It is strongly suggested to use the Galaxy ToolShed method for adding data types.
```

Comment:
That's not the case, we strongly encourage to add datatypes to the galaxy repo, not the Tool Shed.

Reply:
Sorry for this, I meant 'custom' datatypes that are not used outside of an organisation with this comment. Of course for datatypes that are used by certain tools, these should definitely be added to the Galaxy repo.

Reply:
I'm removing it from this.